src/frontend/shortlist/src/components/layout/signup_login/__tests__/LoginSignup.test.js

```diff
@@ -0,0 +1,317 @@
+    });
+    const logoToHome = componentWrapper.find("#logoToHome");
+    expect(logoToHome.exists()).toBe(true);
+  });
+
+  it("Check if the logo_image_container exists", async () => {
+    const componentWrapper = mount(Login, {
+      props: {},
+      global: {
+        plugins: [createTestingPinia({ createSpy: vi.fn })],
+      },
+    });
+    const logo_image_container = componentWrapper.find("#logo_image_container");
+    expect(logo_image_container.exists()).toBe(true);
+  });
+
+  it("Check if the logoImage exists", async () => {
+    const componentWrapper = mount(Login, {
+      props: {},
+      global: {
+        plugins: [createTestingPinia({ createSpy: vi.fn })],
+      },
+    });
+    const logoImage = componentWrapper.find("#logo_img");
+    expect(logoImage.exists()).toBe(true);
+  });
+});
+
+describe("LoggedOut.vue", () => {
+  it("Renders with null props", () => {
+    const componentWrapper = mount(LoggedOut, {
+      props: {},
+      global: {
+        plugins: [createTestingPinia({ createSpy: vi.fn })],
+      },
+    });
+    expect(componentWrapper.exists()).toBe(true);
+  });
+
+  it("Check if container exists", () => {
+    const componentWrapper = mount(LoggedOut, {
+      props: {},
+      global: {
+        plugins: [createTestingPinia({ createSpy: vi.fn })],
+      },
+    });
+    const container = componentWrapper.find(".container");
+    expect(container.exists()).toBe(true);
+  });
+});
+
+describe("ModalFullScreen.vue", () => {
+  // Existence
+  it("Check if ModalFullScreen exists", () => {
+    const componentWrapper = mount(ModalFullScreen, {
+      props: {
+        visible: true,
+      },
+      global: {
+        plugins: [createTestingPinia({ createSpy: vi.fn })],
+      },
+    });
+    expect(componentWrapper.exists()).toBe(true);
+  });
+
+  it("Check if shortlist_modal works", () => {
+    const componentWrapper = mount(ModalFullScreen, {
+      props: {
+        visible: true,
+      },
+      global: {
+        plugins: [createTestingPinia({ createSpy: vi.fn })],
+      },
+    });
+    const shortlist_modal = componentWrapper.find(".shortlist-modal");
+    expect(shortlist_modal.exists()).toBe(true);
+  });
+
+  it("Check if shortlist_modal_inner works", () => {
+    const componentWrapper = mount(ModalFullScreen, {
+      props: {
+        visible: true,
+      },
+      global: {
+        plugins: [createTestingPinia({ createSpy: vi.fn })],
+      },
+    });
+    const shortlist_modal_inner = componentWrapper.find(
+      ".shortlist-modal-inner"
+    );
+    expect(shortlist_modal_inner.exists()).toBe(true);
+  });
+
+  it("Check if stopPropagation works", () => {
+    const componentWrapper = mount(ModalFullScreen, {
+      props: {
+        visible: true,
+      },
+      global: {
+        plugins: [createTestingPinia({ createSpy: vi.fn })],
+      },
+    });
+    const stopPropagation = componentWrapper.find("#stopPropagation");
+    expect(stopPropagation.exists()).toBe(true);
```

Comment:
As mentioned above there are too many existance check for components here, would you also include Daniel in review so we could take a look and see if these check would increase coverage?

Reply:
If you think he can explain better, please feel free to add him